packages/api/src/controllers/auth/v1/mod.rs

```diff
@@ -216,6 +218,83 @@ impl UserControllerV1 {
         todo!()
     }
 
+    pub async fn user_login(&self, body: UserUserLoginRequest) -> Result<JsonWithHeaders<User>> {
```

Comment:
```suggestion
    pub async fn user_login(&self, _auth: Option<Authorization>,  body: UserUserLoginRequest) -> Result<JsonWithHeaders<User>> {
```

get authorization extension from handler.